ynput/OpenPype
# Max: Implement Validator for Properties/Attributes Value Check

## Changelog Description
Add optional validator which can check if the property attributes are valid in Max

## Additional info
Paragraphs of text giving context of additional technical information or code examples.

## Testing notes:
1. Launch Max via launcher
2. Enable Validate Attributes in Settings
3. Put the relevant attributes into the "Attributes" dict. if you want to check if `renderers.current.separateAovFiles=true` or `renderers.current.MotionBlurShutterAngle=160.5` etc..
```
{
    "renderers.current": {
        "separateAovFiles": true,
        "MotionBlurShutterAngle": 160.5,
        "PrimaryGIEngine": "#RS_GIENGINE_BRUTE_FORCE",
        "OCIOViewName": "ACES 1.0 SDR-video"
    }
}
```
![image](https://github.com/ynput/OpenPype/assets/64118225/61f63c87-0af2-4530-a864-db74c11c9b87)

4. Create Instance
5. Publish
6. If it doesn't pass the validation, please check the message to see if it can be fixed by repair action.
7. If you receive Unknown Property Values :{your input attribute}, you need to make sure this attribute exists in maxscript as the property value
8. If the message in 7 doesn't exists, you can use the repair action to repair the setting in 3dsMax.
